.github/workflows/test.yml

```diff
@@ -0,0 +1,69 @@
+name: Tests
+on: [pull_request]
+env:
+  CODE_COVERAGE_THRESHOLD_REQUIRED: 33
```

Comment:
Can we bump this to the best current level?

Reply:
Accomplished.